pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -322,35 +319,27 @@ def on_pretrain_routine_start(self, trainer, pl_module):
 
     @rank_zero_only
     def on_validation_end(self, trainer, pl_module):
-        # only run on main process
-        if trainer.global_rank != 0:
-            return
-
-        # no models are saved
-        if self.save_top_k == 0:
-            return
-
-        if trainer.running_sanity_check:
-            return
-
         metrics = trainer.logger_connector.callback_metrics
         epoch = trainer.current_epoch
 
-        # validate metric
-        if not self._is_valid_monitor_key(metrics):
-            keys = list(metrics.keys())
-            m = f"""
-                ModelCheckpoint(monitor='{self.monitor}') not found in the returned metrics ({keys}),
```

Comment:
""" blocks should not be used for logged messages because they include all the extra whitespace